packages/govuk-frontend-review/src/stylesheets/app.scss

```diff
@@ -4,7 +4,7 @@ $govuk-new-organisation-colours: true;
 
 $govuk-suppressed-warnings: ("organisation-colours");
 
-@import "govuk/all";
+@import "govuk/index";
```

Comment:
**suggestion** I think we can reduce it to `@import "govuk"`, which was one of the motivations of the move to `index` instead of `all`.